src/org/org.ts

```diff
@@ -755,6 +764,90 @@ export class Org extends AsyncOptionalCreatable<Org.Options> {
     );
   }
 
+  /**
+   * Query SandboxInfo Features by sandbox name
+   *
+   * @param sandboxName The sandbox name to query for
+   */
+  public async querySandboxInfoFeaturesBySandboxName(name: string): Promise<string | undefined> {
+    return (
+      await this.connection.singleRecordQuery<{ Features: string }>(
+        `SELECT Features FROM SandboxInfo WHERE SandboxName = '${name}'`,
+        { tooling: true }
+      )
+    ).Features;
+  }
+
+  /**
+   * Query SandboxInfo Features by ID
+   *
+   * @param id The SandboxInfo ID to query for
+   */
+  public async querySandboxInfoFeaturesById(id: string): Promise<string | undefined> {
+    return (
+      await this.connection.singleRecordQuery<{ Features: string }>(
+        `SELECT Features FROM SandboxInfo WHERE Id = '${id}'`,
+        { tooling: true }
+      )
+    ).Features;
+  }
+
+  public async querySandboxInfo(by: SandboxInfoQueryFields): Promise<SandboxProcessObject> {
+    const whereClause = by.id ? `SandboxInfoId='${by.id}'` : `SandboxName='${by.name}'`;
+    const soql = `SELECT ${sandboxProcessFields.join(
+      ','
+    )} FROM SandboxProcess WHERE ${whereClause} ORDER BY CreatedDate DESC`;
+    const result = (await this.connection.tooling.query<SandboxProcessObject>(soql)).records.filter(
+      (item) => !item.Status.startsWith('Del')
+    );
+
+    if (result.length === 0) {
+      throw new SfError(`No record found for ${soql}`, SingleRecordQueryErrors.NoRecords);
+    }
+    if (result.length > 1) {
+      const err = new SfError('The query returned more than 1 record', SingleRecordQueryErrors.MultipleRecords);
+      err.data = result;
+      throw err;
+    }
+    return result[0];
+  }
+
+  /**
+   * Get Features for a sandbox by name or ID, checking both SandboxProcess and SandboxInfo
+   *
+   * @param sandboxName The sandbox name or ID to query for
+   */
+  public async getSandboxFeatures(sandboxName: string): Promise<string | undefined> {
```

Comment:
This is too specific to have on the Org class and should be removed.  The only function to add should be `querySandboxInfo`